src/components/SellingOverviewArticle.css

```diff
@@ -0,0 +1,18 @@
+.TVContainer{
```

Comment:
Volgens de aandachtspuntenlijst is het beter om hier kleine letters te gebruiken.` .tv-container` of zoiets.